### ISSUE TITLE
Alt text is not descriptive [gen2 pages]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7065]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/gen2/deploy-and-host/fullstack-branching/monorepos/index.mdx:
```
  52  
  53: ![monorepo](/images/gen2/fullstack-branching/monorepo.png)
  54  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/gen2/deploy-and-host/sandbox-environments/setup/index.mdx:
```
  21  
  22: ![How cloud sandbox environments work](/images/gen2/how-amplify-works/sandbox.png)
  23  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/gen2/how-amplify-works/concepts/index.mdx:
```
  130  
  131: ![Screenshot showing Figma to Code](/images/gen2/how-amplify-works/ui.jpg)
  132  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/gen2/start/mobile-support/index.mdx:
```
  872  
  873: ![Shows the project details dialog](/images/lib/getting-started/ios/set-up-swift-3.png)
  874  
```
